django/thunderstore/legal/templates/contracts/contract.html

```diff
@@ -0,0 +1,52 @@
+{% extends 'base.html' %}
+{% load markdownify %}
+{% load cache %}
+
+{% block title %}{{ contract.title }}{% endblock %}
+
+{% block main_content_left %}{% endblock %}
```

Comment:
Are these four here just to override some defaults with blanks? Same applies to `contract_history.html`.

Reply:
Yep